bash_completion

```diff
@@ -398,6 +415,7 @@ _comp_split()
 #   -c cur  Set a word used as a prefix to filter the completions.  The default
 #           is ${cur-}.
 #   -R      The same as -c ''.  Use raw outputs without filtering.
+#   -C dir  Evaluate compgen/generator in the specified directory.
```

Comment:
https://github.com/scop/bash-completion/pull/847/files#diff-f9cb750034235bcb02637f30a2708708b7d5c6bb3df32c26dfa8312effc6b1f2 has some tests I suppose we could include/port here.

Reply:
I have ported the tests and also added other two tests. The slash on this [line](https://github.com/scop/bash-completion/pull/847/files#diff-f9cb750034235bcb02637f30a2708708b7d5c6bb3df32c26dfa8312effc6b1f2R175) has been dropped in the ported version because as discussed in [#847 (comment)](https://github.com/scop/bash-completion/pull/847/files#r1028561620), `-o filenames` will not suffix a slash to filenames that do not exist in the current directory.